api/v1/models/Hacker.js

```diff
@@ -1,11 +1,13 @@
 /* jshint esversion: 6 */
 
-var registration = require('../utils/registration');
+var utils = require('../utils');
+var registration = utils.registration;
 var _ = require('lodash');
 
 var OMITTED_ATTRIBUTES = [];
```

Comment:
Is this being used? It seems like this should be a constant either way, so just use the ES6 `const` keyword in place of `var`.